src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java

```diff
@@ -117,11 +115,8 @@ public TestStreamingClient start() {
 
     public boolean close() {
         if (running) {
-            try {
-                connection.disconnect();
-            } finally {
-                running = false;
-            }
+            running = false;
+            connection.disconnect();
```

Comment:
I think he already pushed it to master, but it did not help. The `connection` is the instance of the `HttpUrlConnection`. Is that what you meant?